### PR TITLE
Update gsub regexs

### PR DESCRIPTION
**Pull request by schneiderderek**
This was branched off of https://github.com/covermymeds/sql_footprint/pull/7. So only look at commits after 33a4550.

### CODE REVIEW

**Comment by schneiderderek** [comment]
@mikegee Generalizing the number regex has the side effect of redacting queries using limits. See: https://github.com/covermymeds/sql_footprint/pull/10/commits/7cb7b8317800783773e1274c7cf645eebc213148#diff-ed3453b1981b0fcee6798baf1b2b56d4R34

**Comment by mikegee** [comment]
Well, that seems bad.

**Comment by schneiderderek** [comment]
@mikegee we could have a list of operators that can be used with a number and just list those.

File: lib/sql_footprint/sql_anonymizer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module SqlFootprint
+  class SqlAnonymizer
+    GSUBS = {
+      /\s+IN\s+\(.*\)/ => ' IN (values-redacted)'.freeze, # IN clauses
+      /\s+[0-9]+/ => ' number-redacted'.freeze, # numbers
```
Comment (brandonjoyce) [inline lib/sql_footprint/sql_anonymizer.rb]: I was anchoring w/ the '=' (and now just operator list) because tables and columns can be named as numbers.

Comment (brandonjoyce) [inline lib/sql_footprint/sql_anonymizer.rb]: Maybe we should write a test for that to make it clear though.

**Comment by schneiderderek** [comment]
some of this PR has been addressed by others.

**Pull request closed without merging**